runner-config.md

```diff
@@ -0,0 +1,44 @@
+To run the tests with a customized configuration, you can add a new setting with key named: `java.test.config`.
+
+> [How to create new settings in VS Code?](https://code.visualstudio.com/docs/getstarted/settings#_creating-user-and-workspace-settings)
+
+## TL;DR
+Here is an example of the configuration schema:
+```js
+"java.test.config": [
+    {
+        "name": "myConfiguration"
+        "workingDirectory": "${workspaceFolder}"
+        "args": [ "-c", "com.test" ],
+        "vmargs": [ "-Xmx512M" ],
+        "env": { "key": "value" },
+    },
+    {
+        // Another configuration item...
+    },
+  ...
+]
+```
+
+> Note: The first configuration item in the array will be used as the default configuration when you trigger `Run Test` or `Debug Test`
+
+## Property Details
+
+The value of `java.test.config` is an object array, each object is a configuration item, which may have following properties:
+| Property Name | Description | Default Value |
+|---|---|---|
+| `name` | Specify the name of the configuration item | "" |
+| `workingDirectory` | Specify the working directory when running the tests | "${workspaceFolder}" |
+| `vmargs` | Specify the extra options and system properties for the JVM | [] |
+| `args` | Specify the command line arguments which will be passed to the test runner | [] |
+| `env` | Specify the extra environment variables when running the tests | {} |
+
+## Q & A
+**Q: How can I migrate to the new `java.test.config` setting?**
+A: You can create your own customized configuration according to the above document. Then remove or rename the `launch.test.json` in your workspace.
```

Comment:
customized configuration -> test configuration using the instruction above

Then the `launch.test.json` can be removed.